### PR TITLE
Implement selectors

### PR DESCRIPTION
**Pull request by marc-costello**
- Adds a evt token attribute to every element which passes through.
- Return a new evt object with the element(s) they passed in.
- Basic build task

This is all basic, and I haven't even tested to see if this works. Looks okay on the face of it. Could probably do with some refactoring to pull some of the work out - for now though it's not hurting anybody.

### CODE REVIEW

File: src/evt.js [hunk 1]
```diff
@@ -36,11 +28,29 @@ evt.prototype.raise = function() {
...
 
 function evtInit(element) {
-
-  // handle the element/selector
-
-  return new evt();
+  if (!element || (element instanceof Element) === false) {
+    throw new Error('You must supply a DOM element or selector to evt');
+  }
+
```
Comment (richarddowson) [inline src/evt.js:34]: If element is a selector, it's going to need to be a string. If not, it should be an Element, or a collection of Elements.

Comment (marc-costello) [inline src/evt.js:34]: Yea could probably check for a string.

I already check for an element.

Do we need to accept and array of elements? Does jquery do that?

Comment (richarddowson) [inline src/evt.js:34]: jquery accepts a selector or an element I think, but not an array. The trouble is that querySelectorAll returns a NodeList, which isn't actually an array. If we accept a collection of elements, it should be an array or a NodeList; that way the user can simply pass in the result of querySelectorAll. I reckon for now, let's keep it to single Element, or a string selector. We can add collections later if need be.

Comment (marc-costello) [inline src/evt.js:34]: So isn't that how I've done it? Except there needs a check for string.
Then you cant pass any thing other than a string or a instance of Element. Don't think node list is an element.

Comment (richarddowson) [inline src/evt.js:34]: Apologies, yeah. Just needs the string check. The whole collection thing is a separate issue...

**Pull request merged**